programs/manifest/src/quantities.rs

```diff
@@ -490,24 +460,24 @@ fn calculate_mantissa(value: f64, exp: i8) -> f64 {
 }
 
 impl TryFrom<f64> for QuoteAtomsPerBaseAtom {
-    type Error = PriceConversionError;
+    type Error = ManifestError;
```

Comment:
bench perf for this change

Reply:
<img width="1071" alt="Screenshot 2024-10-06 at 7 13 53 PM" src="https://github.com/user-attachments/assets/1eb17650-3371-41ea-aa37-b6638fec62d4">
within noise